pkg/webhook/mutation.go

```diff
@@ -63,6 +64,7 @@ func AddMutatingWebhook(mgr manager.Manager, deps Dependencies) error {
 	eventBroadcaster := record.NewBroadcaster()
 	kubeClient := kubernetes.NewForConfigOrDie(mgr.GetConfig())
 
+	log := log.WithValues("hookType", "mutation")
```

Comment:
nit: do we want to add `log.WithName("mutation").WithValues...` to group the same type of logs together?

Reply:
I can pull these out and have them init-ed at runtime as 

```
pkg/webhook/mutation.go

var mutationLog = common.log.WithValues("hookType", "mutation")
```

and have that given to the handler. Would that work? Or was there a different place where I should put the `mutationLog`?

Also, just to clarify, the base `log.` on the RHS is defined in common.go:

https://github.com/open-policy-agent/gatekeeper/blob/ba39189a1b298c931a615bee59342794414a4161/pkg/webhook/common.go#L39

I think we should keep the `logname` as `webhook` to not break people scraping the logs. People can still group the logs by looking for `hookType = mutation` or `hookType = validation`. Wdyt?

Reply:
> the base log. on the RHS is defined in common.go
Ah ok. I missed that. then yes it already has the `webhook` name, we should use the same.